src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -355,7 +355,7 @@ def remove_host(self, host: str, force: bool, offline: bool) -> OrchResult[str]:
         """
         raise NotImplementedError()
 
-    def drain_host(self, hostname: str) -> OrchResult[str]:
+    def drain_host(self, hostname: str, force: bool) -> OrchResult[str]:
```

Comment:
@adk3798 I think this broke the API call from dashboard. That's why cephadm e2e failed. So can we default the `force` to `False`?
```suggestion
    def drain_host(self, hostname: str, force: bool = False) -> OrchResult[str]:
```

Reply:
done